src/Translator/src/DependencyInjection/UxTranslatorExtension.php

```diff
@@ -43,12 +43,12 @@ public function prepend(ContainerBuilder $container)
         if (!$this->isAssetMapperAvailable($container)) {
             return;
         }
-
+        $config = $container->getExtensionConfig('ux_translator')[0];
         $container->prependExtensionConfig('framework', [
             'asset_mapper' => [
                 'paths' => [
                     __DIR__.'/../../assets/dist' => '@symfony/ux-translator',
-                    '%kernel.project_dir%/var/translations' => 'var/translations',
+                    $config['dump_directory'] => '@app/translations',
```

Comment:
Could you use a functionnal test for this ? That will prevent any future regression

Reply:
Not sure how to test for this. I don't think I can access the config, and I can't functionally test this without requiring asset mapper in require-dev, which is not wanted. If you have any input on how I can retrieve the config from the test scope I'd be glad to hear about it :)

Reply:
I shouldn't have ignored this conversation 😭